### PR TITLE
WIP: rework example app

### PR DESCRIPTION
**Pull request by puckey**
This is still very early work in progress. Comments are welcome in #599

### CODE REVIEW

**Comment by dcvz** [comment]
Hi @puckey you might want to take a look at the latest on master. I've updated the example app to RN 0.59.8 which might make your efforts here a lot easier :)

**Comment by puckey** [comment]
> Hi @puckey you might want to take a look at the latest on master. I've updated the example app to RN 0.59.8 which might make your efforts here a lot easier :)

I had already done the update to RN 0.59.8 in the PR, but thanks – I will take a look at what the differences are

File: example/android/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,5 @@
-#Wed Oct 03 00:44:36 CEST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
```
Comment (aclement-clevertech) [inline example/android/gradle/wrapper/gradle-wrapper.properties]: Might want to use `4.10.3`

File: example/react/components/ProgressBar.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
```
Comment (aclement-clevertech) [inline example/react/components/ProgressBar.js]: `useState` doesn't seems to be used in this file.

**Pull request closed without merging**